### PR TITLE
Add a comment about dummy tokens used in auth tests

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
This PR adds a comment/note to say the tokens/credentials used in auth related tests are just dummy tokens/credentials.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by ldclakmal** [comment]
Closed and reopened due to GitHub actions failure.

File: tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2-mock-server.bal [hunk 1]
```diff
@@ -14,6 +14,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (praneesha) [inline tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2-mock-server.bal:17]:
```suggestion
// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2-mock-server.bal [hunk 2]
```diff
@@ -14,6 +14,8 @@
 // specific language governing permissions and limitations
```
Comment (praneesha) [inline tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2-mock-server.bal:14]:
```suggestion
// specific language-governing permissions and limitations
```

File: tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2_client_test.bal [hunk 3]
```diff
@@ -18,6 +18,8 @@ import ballerina/http;
 import ballerina/config;
 import ballerina/oauth2;
 
+// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (praneesha) [inline tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2_client_test.bal:21]:
```suggestion
// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2_service_test.bal [hunk 4]
```diff
@@ -14,6 +14,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (praneesha) [inline tests/jballerina-integration-test/src/test/resources/auth/src/oauth2/oauth2_service_test.bal:17]:
```suggestion
// NOTE: All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnConfigInheritanceTest.java [hunk 5]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for authentication config inheritance scenarios.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnConfigInheritanceTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnConfigPatternTest.java [hunk 6]
```diff
@@ -34,6 +34,8 @@
  * Pattern 3 - authHandlers: [[basicAuthHandler1]]
  * Pattern 4 - authHandlers: [[basicAuthHandler1], [basicAuthHandler3]]
  * Pattern 5 - authHandlers: [[basicAuthHandler1, basicAuthHandler2], [basicAuthHandler3, basicAuthHandler4]]
+ *
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnConfigPatternTest.java:38]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnWithoutHttpAnnotationsTest.java [hunk 7]
```diff
@@ -31,6 +31,7 @@
 /**
  * Test cases for service level authentication and authorization when service and resource do not have config
  * annotations attached.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthnWithoutHttpAnnotationsTest.java:34]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthzConfigInheritanceTest.java [hunk 8]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for authorization config inheritance scenarios.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthzConfigInheritanceTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthzConfigPatternTest.java [hunk 9]
```diff
@@ -34,6 +34,8 @@
  * Pattern 3 - scopes: [["scope1"]]
  * Pattern 4 - scopes: [["scope1"], ["scope3"]]
  * Pattern 5 - scopes: [["scope1", "scope2"], ["scope3", "scope4"]]
+ *
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/AuthzConfigPatternTest.java:38]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/ResourceLevelAuthTest.java [hunk 10]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for resource level authentication/authorization.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/ResourceLevelAuthTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/ServiceLevelAuthnTest.java [hunk 11]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for service level authentication/authorization.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/basic/ServiceLevelAuthnTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/ldap/LdapAuthStoreTest.java [hunk 12]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for LDAP user store based authentication/authorization scenarios.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/ldap/LdapAuthStoreTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/oauth2/OAuth2ServiceTest.java [hunk 13]
```diff
@@ -28,6 +28,7 @@
 
 /**
  * Test cases for inbound authentication with OAuth2.
+ * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes only.
```
Comment (ldclakmal) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/oauth2/OAuth2ServiceTest.java:31]:
```suggestion
 * All the tokens/credentials used in this test are dummy tokens/credentials and used only for testing purposes.
```

**Pull request merged**